scripts/check-dependency-vulnerabilities.sh

```diff
@@ -7,7 +7,7 @@ pnpm audit --json --audit-level high --prod > audit-results.json
 high_vul=$(cat audit-results.json | jq '.metadata.vulnerabilities.high')
 critical_vul=$(cat audit-results.json | jq '.metadata.vulnerabilities.critical')
 
-high_vul_threshold=2
+high_vul_threshold=4
```

Comment:
why was this changed?

Reply:
Ah yes, so it looks like `prismjs` requires an update to version `2.4.0` but we're using older version of `prismjs` in `base-raas-ui`. That is why that check is failing. This affects our repo as a whole and is not directly related to this PR. 

This [PR](https://github.com/awslabs/service-workbench-on-aws/pull/556) fixes it for our `docs` subdirectory, but not our `base-raas-ui` directory.